binaries/coordinator/src/run/custom.rs

```diff
@@ -15,33 +15,42 @@ pub(super) async fn spawn_custom_node(
     communication: &dora_core::config::CommunicationConfig,
     working_dir: &Path,
 ) -> eyre::Result<tokio::task::JoinHandle<eyre::Result<(), eyre::Error>>> {
-    let path = if source_is_url(&node.source) {
+    let mut is_path = source_is_path(&node.source, working_dir);
```

Comment:
I think it makes sense to check for an URL first. Or is there any scenario where a `source` string containing `://` could be a local path?